umap/static/umap/js/modules/urls.js

```diff
@@ -0,0 +1,29 @@
+import { Util } from './vendors.js'
```

Comment:
I'm not fan of this import, as we lose the origin of the `Util` variable in the process.

I'd suggest, either:
- loading `Util` (and such) from `path/to/leaflet.esm.js` everywhere
- exposing namespace in vendors, not variables; so one would type `import L from './vendors` and then use `L.Util`, just like in classic mode

Reply:
Yep, I like the second one better, and I agree 👍